docker/run.sh

```diff
@@ -1,9 +1,41 @@
 #!/usr/bin/env bash
 
+set -Eeuo pipefail
+
 # Run nginx as root
 sed -i 's/user  nginx.*$//' /etc/nginx/nginx.conf
 
 # Write selected env vars to nginx config
 envsubst '$PORT' < /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf
 
+# Inject runtime config into the client
+function inject_config() {
```

Comment:
Uhhh can we at this point make this entire file into a node script? 😛

Reply:
Nah, no node shipped in this container :grin:

Imo it's much nicer to have a piece of bash with a lightweight dependency like `jq` to copy around rather than forcing every container to have node installed.